test/scripts/e2e.sh

```diff
@@ -158,7 +158,26 @@ if [ -z "$E2E_TEST_FILTER" ] || [ "$E2E_TEST_FILTER" == "SCRIPTS" ]; then
     ./timeout 200 ./e2e_basic_start_stop.sh
     duration "e2e_basic_start_stop.sh"
 
-    "${TEMPDIR}/ve/bin/python3" e2e_client_runner.py ${RUN_KMD_WITH_UNSAFE_SCRYPT} "$SRCROOT"/test/scripts/e2e_subs/*.{sh,py}
+    if [ $E2E_PLATFORM == "arm64" ]; then
+	echo "Keeping temporary running"
+	"${TEMPDIR}/ve/bin/python3" e2e_client_runner.py --keep-temps ${RUN_KMD_WITH_UNSAFE_SCRYPT} "$SRCROOT"/test/scripts/e2e_subs/*.{sh,py}
+
+	pushd "${TEMPDIR}" || exit 1
+
+	aws configure list
+
+	tar -j -c -f net_done.tar.bz2 --exclude node.log --exclude agreement.cdv net
+	rm -rf "${TEMPDIR}/net"
+	RSTAMP=$(TZ=UTC python -c 'import time; print("{:08x}".format(0xffffffff - int(time.time() - time.mktime((2020,1,1,0,0,0,-1,-1,-1)))))')
+	echo aws s3 cp --acl public-read "${TEMPDIR}/net_done.tar.bz2" s3://algorand-testdata/indexer/e2e4/"${RSTAMP}"/net_done.tar.bz2
+	aws s3 cp "${TEMPDIR}/net_done.tar.bz2" s3://algorand-testdata/indexer/e2e4/"${RSTAMP}"/net_done.tar.bz2
+#	aws s3 cp "${TEMPDIR}/net_done.tar.bz2" s3://algorand-testdata-sandbox/indexer/e2e4/"${RSTAMP}"/net_done.tar.bz2
+	popd
+
+    else
+	"${TEMPDIR}/ve/bin/python3" e2e_client_runner.py ${RUN_KMD_WITH_UNSAFE_SCRYPT} "$SRCROOT"/test/scripts/e2e_subs/*.{sh,py}
+    fi
```

Comment:
Use spaces instead of tabs.

Remove commented out test.

Add a comment about why we are using --keep-temps, and that the artifact is used by Indexer.

Could you adjust the code to call `e2e_client_runner.py` once instead of with an if/else? Pass `--keep-temps` as a variable like we do with `${RUN_KMD_WITH_UNSAFE_SCRYPT}`. This way if we change e2e_client_runner in the future, we don't need to change as many places (i.e. in the lines below we still have `*.sh`, which might be wrong).

After the job finishes, you can check if `"${TEMPDIR}/net"` exists and use that to trigger the upload.